<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appHandleWidget</class>
 <widget class="QWidget" name="appHandleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>app操作</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,1,5">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>100</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>appName:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="appNameComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>envName:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="envNameComboBox"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>60</number>
       </attribute>
       <column>
        <property name="text">
         <string>应用名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>环境名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>git版本号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>操作</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
